src/main/java/com/zendesk/maxwell/row/RowMapBuffer.java

```diff
@@ -1,10 +1,13 @@
 package com.zendesk.maxwell.row;
 
 import com.zendesk.maxwell.util.ListWithDiskBuffer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 
 public class RowMapBuffer extends ListWithDiskBuffer<RowMap> {
```

Comment:
@lukestephenson-zendesk It's correct... enough.  But the whole thing appears to need a cleanup; the superclass, ListWithDiskBuffer (of which we are of course the only consumers), calculates based on nElements, but RowMapBuffer mostly makes its own calculates based on RSS estimates. 

I think it's been long enough with no other consumers of `ListWithDiskBuffer` that we could just merge it and `RowMapBuffer` and stop ever talking about max-elements (instead always trying to talk about estimated-rss)

Reply:
It's an override, it attempts to get a more accurate result by measuring actual bytes instead of just rows.

It also overrides `evict()` to apply a similar logic, since that's the other place `maxInMemoryElements` is used.

So maxInMemoryElements (mostly) has no effect. Also, RowMapBuffer is the only concrete instance of ListWithDiskBuffer, so we should probably just merge them or detach through a proper interface, because understanding what actually happens with this inheritance is tough.

Reply:
(i.e. "ditto" - didn't see Ben's comment when I posted that)

Reply:
> ok, but `LOGGER.info` is way too high a default here. you'll have to go down to debug and deploy a custom log4j2.xml.

Yep, that makes sense. I'll change it to debug for the os cache resetting info. 

I'd like to leave the startup logging as info though as it is only displaying configuration values.